### PR TITLE
Elegoo 3 plus fix

### PR DESCRIPTION
**Pull request by GregValiant**
# Description

There were typos in the Neptune 3Plus nozzle files.

## Type of change

- [ X] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
- [ X] Test A - The fix adds the nozzles to the 2Plus printer

**Test Configuration**:
* Operating System:  Windows 10

- [ X] My code follows the style guidelines of this project as described in [UltiMaker Meta](https://github.com/Ultimaker/Meta) and [Cura QML best practices](https://github.com/Ultimaker/Cura/wiki/QML-Best-Practices)
- [X ] I have read the [Contribution guide](https://github.com/Ultimaker/Cura/blob/main/contributing.md) 
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ X] I have uploaded any files required to test this change

### CODE REVIEW

**Comment by MariMakes** [comment]
Hey @GregValiant,

Good to see you! Thanks for the PR!
It's the typo mentioned, [here](https://community.ultimaker.com/topic/43732-ultimaker-cura-54-beta-released/), right?

**Comment by GregValiant** [comment]
That's it.  Right file but wrong definition.

File: plugins/PostProcessingPlugin/scripts/ExampleScript.md [hunk 1]
```diff
@@ -1,3 +0,0 @@
-A good example script is SearchAndReplace.py.
-If you have any questions please ask them at: 
-https://github.com/Ultimaker/Cura/issues
```
Comment (casperlamboo) [inline plugins/PostProcessingPlugin/scripts/ExampleScript.md:3]: Although this file is a bit non-sense, I don't think I should be removed in this PR.

**Review by casperlamboo — CHANGES_REQUESTED** [review]
Thanks @GregValiant , good catch!

If you can revert the changes on `ExampleScript` I will merge the PR.

**Comment by GregValiant** [comment]
Hi Casper.
It's true...I suck at pull requests.  That wasn't intended to be part of it and I have no idea how it got in there.

**Comment by casperlamboo** [comment]
Cherry picked the relevant changes directly to the 5.4 branch https://github.com/Ultimaker/Cura/commit/8b7339e399e69644a1364766e921ce877c567c42. Pushing that commit in favor of the PR; closing.

Thank you @GregValiant for the fix!

**Comment by GregValiant** [comment]
Perfect.

**Pull request closed without merging**